src/api/index.ts

```diff
@@ -178,9 +178,19 @@ export async function startUserModule(id: string) {
   return handleResponse(await api.post<UserModule>(`/user-module/${id}`)).data;
 }
 
+
 export async function verifyEmail(code: string) {
   return handleResponse(await api.post<string>(`/user/verify-email`, {code: code}));
 }
+
+export async function verifyUser() {
+  return handleResponse(await api.get<boolean>(`/user/verify-user`)).data;
+}
+
+export async function resendEmail() {
```

Comment:
@jasekiw  Correct me if I am wrong.
This should be renamed to resendVerifyUserEmail as we will be sending other types of emails in the future.